type_description_interfaces/package.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>type_description_interfaces</name>
+  <version>1.3.1</version>
+  <description>A package containing message and service definitions for describing and communicating descriptions of other types.</description>
+
+  <maintainer email="william@openrobotics.org">William Woodall</maintainer>
+
+  <license>Apache License 2.0</license>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
+  <buildtool_depend>rosidl_core_generators</buildtool_depend>
+
+  <exec_depend>rosidl_core_runtime</exec_depend>
+
+  <test_depend>ament_lint_common</test_depend>
```

Comment:
```suggestion
  <test_depend>ament_lint_auto</test_depend>
  <test_depend>ament_lint_common</test_depend>
```

Reply:
Added in 2563dc382581359b57d945b3a8b321599e215030